src/graph.js

```diff
@@ -60,6 +61,19 @@ module.exports = (function () {
         return this;
     };
 
+    Graph.prototype.dontRender = function() {
+        this._renderNow = false;
+
+        return this;
+    };
+
+    Graph.prototype.doRender = function() {
```

Comment:
Works for me. How should I go about running those funcs during the bulk adding? As is (i.e. `disableAutoRender` before the forEach, `enableAutoRender` after), or let the user decide? Having taken it for a spin today I'm kinda getting convinced about going for the latter but, again, your call.

Also, are you cool with me doing this as a separate commit (for history's sake), or shall I partake in squash/amend-related shenanigans?